Game/battle.py

```diff
@@ -0,0 +1,264 @@
+				trip = Trip(character, enemy)
+				trip.action()
+				hitpointCheck(character, enemy, room)
+			else:
+				print "Try something different.\n"
+
+	if character.type == "thief":
+		loopMove = False
+		while loopMove == False:
+			print "You decide to fight, what move do you perform?\n"
+			print "Do you:\n"
+			print "\t1. Stab"
+			print "\t2. Trip"
+
+			move = raw_input("> ")
+
+			if move in ("1", "Stab", "stab"):
+				loopMove = True
+				move = Stab(character, enemy)
+				move.action()
+				hitpointCheck(character, enemy, room)
+			elif move in ("2", "Trip", "trip"):
+				loopMove = True
+				trip = Trip(character, enemy)
+				trip.action()
+				hitpointCheck(character, enemy, room)
+			else:
+				print "Try something different.\n"
+
+
+class Swing(object):
+
+	def __init__(self, character, enemy):
+		self.character = character
+		self.enemy = enemy
+
+	def action(self):
+		rand = random.randint(0, 1)
+
+		if rand == 1:
+			good = "good"
+			print "You hit the %s with %s!" % (self.enemy.type, self.character.weapon.weapon)
+			battleMath(self.character, self.enemy, good)
+		else:
+			bad = "bad"
+			print "You swing your %s at the %s and you miss!" % (self.character.weapon.weapon, self.enemy.type)
+			print "\nThe enemy %s makes a move at you with his %s" % (self.enemy.type, self.enemy.weapon.weapon)
+			battleMath(self.character, self.enemy, bad)
+
+
+class Stab(object):
+
+	def __init__(self, character, enemy):
+		self.character = character
+		self.enemy = enemy
+
+	def action(self):
+		rand = random.randint(0, 1)
+
+		if rand == 1:
+			good = "good"
+			print "You stab the %s with %s!" % (self.enemy.type, self.character.weapon.weapon)
+			battleMath(self.character, self.enemy, good)
+		else:
+			bad = "bad"
+			print "You stab your %s at the %s and you miss!" % (self.character.weapon.weapon, self.enemy.type)
+			print "\nThe enemy %s makes a move at you with his %s" % (self.enemy.type, self.enemy.weapon.weapon)
+			battleMath(self.character, self.enemy, bad)
+
+
+class Shoot(object):
+
+	def __init__(self, character, enemy):
+		self.character = character
+		self.enemy = enemy
+
+	def action(self):
+		rand = random.randint(0, 1)
+
+		if rand == 1:
+			good = "good"
+			print "You shoot the %s with the %s!" % (self.enemy.type, self.character.weapon.weapon)
+			battleMath(self.character, self.enemy, good)
+		else:
+			bad = "bad"
+			print "You shoot your %s at the %s and you miss!" % (self.character.weapon.weapon, self.enemy.type)
+			print "\nThe enemy %s makes a move at you with his %s" % (self.enemy.type, self.enemy.weapon.weapon)
+			battleMath(self.character, self.enemy, bad)
+
+
+class Trip(object):
+
+	def __init__(self, character, enemy):
+		self.character = character
+		self.enemy = enemy
+
+	def action(self):
+		print "\nYou sweep your foot behind its leg with the utmost of"
+		print "finess and put the %s on its back and continue to attack it." % self.enemy.type
+
+		self.enemy.hitpoints = 0
+
+
+# Will implement this later
```

Comment:
# TODO is a great way to remind yourself it's something you're going to do later rather than comments. Some IDE's and Text Editors can also track them as TODOs :+1: